cmd/vulcan-nuclei/templates/network/exposed-bgp.yaml

```diff
@@ -0,0 +1,26 @@
+id: exposed-bgp
+
+info:
+  name: Exposed BGP Speaker
+  author: Adevinta
+  severity: high
+  description: A potentially exposed BGP port has been found open in the target. According to best practices, BGP ports should not be open to the public internet and access to them should be restricted only to participating BGP neighbours.
+  reference:
+    - https://tools.ietf.org/html/bcp194#section-4
+  tags: network
+
+network:
+  - inputs:
+      - data: ""
+        read: 1024
+
+  - host:
+      - "{{Hostname}}"
```

Comment:
Oh, nice catch! If a port is supplied that port responding to TCP requests would be enough to trigger a detection instead of port 179 as intended.

On addition to this, I think maybe I would just drop this check instead of porting it to Nuclei, what do you think? The reason being that it is just checking for an open port which is not much of a security check in its own right. I think it would be better to group this with the other checks that check for open ports (e.g. exposed databases and exposed router ports) which can also check against the Nmap service fingerprints.

Unless you think we should re-open it, I am closing this PR to work on this check separately.

Reply:
Yeah, it makes a lot of sense. Let's do that! :)